python/triton/language/core.py

```diff
@@ -843,7 +844,7 @@ def dot(input, other, allow_tf32=True, _builder=None):
 
 
 @builtin
-def load(pointer, mask=None, other=None, cache_modifier="", eviction_policy="", volatile=False, _builder=None):
+def load(pointer: tensor, mask: Optional[tensor]=None, other: Optional[tensor]=None, cache_modifier="", eviction_policy="", volatile=False, _builder=None) -> tensor:
```

Comment:
`mask` and `other` here are  passed to `_to_tensor` so I believe their types should instead be `Optional[Tensorable]`